Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -32,40 +47,95 @@ open class WayNameView: UIView {
         }
     }
     
+    open override var layer: CALayer {
+        containerView.layer
+    }
+    
+    /**
+     The background color of the `WayNameView`.
+     */
+    @objc dynamic public override var backgroundColor: UIColor? {
+        get {
+            guard let color = containerView.backgroundColor else { return nil }
+            return color
+        }
+        
+        set {
+            containerView.backgroundColor = newValue
+        }
+    }
+    
+    /**
+     The color of the `WayNameView`'s border.
+     */
     @objc dynamic public var borderColor: UIColor? {
         get {
             guard let color = layer.borderColor else { return nil }
             return UIColor(cgColor: color)
         }
+        
         set {
             layer.borderColor = newValue?.cgColor
         }
     }
     
+    /**
+     The width of the `WayNameView`'s border.
+     */
+    @objc dynamic public var borderWidth: CGFloat {
+        get {
+            layer.borderWidth
+        }
+        
+        set {
+            layer.borderWidth = newValue
+        }
+    }
+    
+    /**
+     Initializes a newly allocated `WayNameView` object with the specified frame rectangle.
+     
+     - parameter frame: The frame rectangle for the `WayNameView`.
+     */
```

Comment:
Quick Help will pick up the comment from `UIView(frame:)`, but jazzy won’t because it doesn’t have access to the Apple documentation. It won’t even link to the Apple documentation: realm/jazzy#13. That said, I don’t think this comment is adding anything that developers would need to know beyond what UIView provides.

Reply:
Makes sense, removed docs from default methods.